src/Test/UnitTests/constraints.jl

```diff
@@ -310,3 +310,51 @@ function solve_affine_deletion_edge_cases(model::MOI.ModelLike, config::TestConf
     )
 end
 unittests["solve_affine_deletion_edge_cases"] = solve_affine_deletion_edge_cases
+
+function solve_zeroone_with_bounds_1(model::MOI.ModelLike, config::TestConfig)
+    MOI.empty!(model)
+    MOIU.loadfromstring!(model,"""
+        variables: x
+        maxobjective: 2.0x
+        c1: x in ZeroOne()
+        c2: x in Interval(0.0, 1.0)
```

Comment:
JuMP doesn't encode variable bounds using `Interval`, so this might not cover the typical code path.